pkg/scheduler/factory.go

```diff
@@ -498,7 +498,7 @@ func MakeDefaultErrorFunc(client clientset.Interface, podLister corelisters.PodL
 				}
 			}
 		} else {
-			klog.Errorf("Error scheduling %v/%v: %v; retrying", pod.Namespace, pod.Name, err)
+			klog.ErrorS(err, "Error scheduling; retrying", "pod", klog.KRef(pod.Namespace, pod.Name))
```

Comment:
```suggestion
			klog.ErrorS(err, "Error scheduling; retrying", "pod", klog.KObj(pod))
```

Reply:
what will format look like when this log message is printed?

Reply:
@tanjunchen You can use instructions provided here https://github.com/kubernetes/community/blob/fc2bfe4c10e901d5d214697a6717c0dc98e6fbd7/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#verify-log-output to generate log output

Reply:
E0607 21:28:51.095352   14016 main.go:71] Error scheduling default/nginx-test: something wrong; retrying
E0607 21:28:51.121282   14016 main.go:73] "Error scheduling; retrying" err="something wrong" pod="default/nginx-test"